### ISSUE TITLE
Apparently netcdf needs to be compiled an additional option

### ISSUE THREAD

**Issue by doutriaux1** [conda-forge/libnetcdf-feedstock#42]
Using 4.4 the following command n used to work (linux)

Using 4.6 I now get:
```
ncdump -h mpaso.hist.am.timeSeriesStatsMonthly.0001-01-01.nc
ERROR: ld.so: object '/lib/i686/nosegneg/libc.so.6' from LD_PRELOAD cannot be preloaded: ignored.
ERROR: ld.so: object '/lib/i686/nosegneg/libc.so.6' from LD_PRELOAD cannot be preloaded: ignored.
ncdump: mpaso.hist.am.timeSeriesStatsMonthly.0001-01-01.nc: NetCDF: Attempt to use feature that was not turned on when netCDF was built.
```

The message is not very informative. I suspect it has to do with very big variables in very big files

The file is ***HUGE*** (~75Gb) you can obtain it at:

```
curl https://cdat.llnl.gov/cdat/sample_data/mpaso.hist.am.timeSeriesStatsMonthly.0001-01-01.nc -o mpaso.hist.am.timeSeriesStatsMonthly.0001-01-01.nc
```

Original issue that triggered the need for the latest netcdf can be seen at: https://github.com/CDAT/cdms/issues/238

**Comment by ocefpaf** [comment]
> Using 4.4 the following command n used to work (linux)

Can you try to force an environment with the old version?

```shell
conda create --name TEST --channel conda-forge libnetcdf=4.4
```

and repeat your test just to be sure it is the new package?

There lines worries me :-/
```
ERROR: ld.so: object '/lib/i686/nosegneg/libc.so.6' from LD_PRELOAD cannot be preloaded: ignored.
ERROR: ld.so: object '/lib/i686/nosegneg/libc.so.6' from LD_PRELOAD cannot be preloaded: ignored.
```

We do rely on system `libc` but I cannot reproduce that error on my Linux machine.

And this line is really not very helpful :unamused: 
```
ncdump: mpaso.hist.am.timeSeriesStatsMonthly.0001-01-01.nc: NetCDF: Attempt to use feature that was not turned on when netCDF was built.
```

@czender does this have anything to do with the bug you mentioned?

**Comment by czender** [comment]
Yes, it does. Because of that bug Unidata changed the default behavior in netCDF 4.5.0, and never re-enabled the old 4.4.x behavior. As of netCDF 4.5.0, please add the configure option "--enable-cdf5". This is something I overlooked and should have noticed earlier. Once netCDF is built with that flag, the above ncdump command should work. Perhaps we could add a test to verify netCDF has been built with CDF5 enabled? `nc-config` can be used for this purpose, e.g.,
```
zender@skyglow:~$ nc-config --has-cdf5
yes
```
I will ask Unidata to make --enable-cdf5 the default again in netCDF 4.7.0, but this option will still be necessary until that occurs.

**Comment by WardF** [comment]
Enabling CDF5 by default is doable at this point, we have ironed out the issues that required it to be turned off in the first place.

**Comment by pmspire** [comment]
CDF5 support seems to have disappeared (in commit ed28c3de55d52912c963384db72e281df167b4cf?)
```
(old) ~ % conda list | grep netcdf
libnetcdf                 4.6.1                         2    conda-forge
(old) ~ % nc-config --has-cdf5
yes
(old) ~ % conda deactivate
~ % conda activate new
(new) ~ % conda list | grep netcdf
libnetcdf                 4.6.1                he6cff42_5    conda-forge
(new) ~ % nc-config --has-cdf5
no
```
Was this intentional?

**Comment by ocefpaf** [comment]
@pmspire that was probably on oversight on my part when I synced the recipes. Do you want to send a PR re-adding that?

**Comment by pmspire** [comment]
I'll try to submit a PR for this today.

**Comment by ocefpaf** [comment]
> I'll try to submit a PR for this today.

B/c of the testing I went ahead and did myself. Thanks for the offer and sorry for asking you to do it and then doing it...

See https://github.com/conda-forge/libnetcdf-feedstock/pull/51

**Comment by pmspire** [comment]
No need to apologize for that! Thanks as always for your tireless conda-forge efforts.
